app/graphql/types/library_entry/status.rb

```diff
@@ -1,7 +1,7 @@
 class Types::LibraryEntry::Status < Types::BaseEnum
-  value 'current', ''
-  value 'planned', ''
-  value 'completed', ''
-  value 'on_hold', ''
-  value 'dropped', ''
+  value 'current', 'Media that you currently read/watch.'
```

Comment:
`that you are currently reading/watching.

Reply:
Although can probably strip `that` from all of these since it's a bit redundant.

Reply:
yeah, you are right, thank you!